common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -350,7 +351,7 @@ def get_html(self):
             'cdn_eval': cdn_eval,
             'cdn_exp_group': cdn_exp_group,
             'id': self.location.html_id(),
-            'display_name': self.display_name_with_default_escaped,
+            'display_name': self.display_name_with_default,
```

Comment:
Yes.  Assuming you mean this line: https://github.com/open-craft/edx-platform/blob/33b3dfcc9279ce59a1b8adf49adb638202d8680f/lms/templates/video.html#L8

It looks like this would have been double-escaped before, and would now be properly single escaped.  I have not tested this, but it looks good.